docs/AcceptableUnicode.md

```diff
@@ -0,0 +1,92 @@
+# Acceptable Unicode characters:
+Using Unicode seems to be irresistible. However, we must ensure
+avoiding problematic Unicode usage. Here is a list of acceptable
+and forbidden Unicode characters and accents. Please note that
+we forbid the use of accents, because this can lead to visually
+ambiguous characters (see below).
+
+## Acceptable lower-case Greek letters:
+α # (alpha)
+β # (beta)
+δ # (delta)
+ϵ # (epsilon)
+γ # (gamma)
+κ # (kappa)
+λ # (lambda)
+μ # (mu)
+η # (eta)
+ω # (omega)
+π # (pi)
+ρ # (rho)
+σ # (sigma)
+θ # (theta)
+χ # (chi)
+ξ # (xi)
+ζ # (zeta)
+ϕ # (psi)
+φ # (varphi)
+
+## Acceptable upper-case Greek letters:
+Δ # (Delta)
+∑ # (Sigma)
+Γ # (Gamma)
+Ω # (Omega)
+Φ # (Phi)
+Ψ # (Psi)
+
+## Acceptable mathematical symbols:
+∫ # (int)
+∬ # (iint)
+∭ # (iiint)
+∈ # (in)
+∞ # (infinity)
+≈ # (approx)
+∂ # (partial)
+∇ # (nabla/del), note that nabla and del are indistinguishable
+∀ # (forall)
+≥ # (greater than equal to)
+≤ # (less than equal to)
+
+## Indistinguishable characters
+Some characters are indistinguishable. For example:
+
+  ∇ = 1 # nabla
```

Comment:
So are nabla and del acceptable? (It's unclear in the document)

Reply:
I would suggest acceptable. I'll make this more clear.

Reply:
Any suggestion about `\nu` and `\upsilon` @jkozdon ?